Add option to use host as Android's DNS server

lwan89 opened the pull request:
Customer may setup their own DNS server which will do some network
access policies. Add this option to CiC, then customer can manage
Android's DNS requests.

Signed-off-by: Wang, Liang <liang.wang@intel.com>

sysopenci reviewed (commented):
Autobuild started from pull-request-changes on this PR.

FAILURE: CheckBug Bad comments/Bugs

For more information, see: /absp/builders/celadon_pmr0_cic-autobuild/builds/235

lwan89 committed:
Add option to use host as Android's DNS server

Customer may setup their own DNS server which will do some network
access policies. Add this option to CiC, then customer can manage
Android's DNS requests.

Tracked-On: OAM-90285
Signed-off-by: Wang, Liang <liang.wang@intel.com>

sysopenci reviewed (commented):
PR integrated into 'preintegration' builder



More details, logs, and binaries: /absp/builders/celadon_pmr0_cic-preintegration/builds/291

Done by "Bai, ShanshanX" <shanshanx.bai@intel.com>

sysopenci reviewed (commented):
PR integrated into 'preintegration' builder



More details, logs, and binaries: /absp/builders/celadon_pmr0_cic-preintegration/builds/290

Done by "Bai, ShanshanX" <shanshanx.bai@intel.com>

sysopenci reviewed (commented):
PR integrated into 'preintegration' builder



More details, logs, and binaries: /absp/builders/celadon_pmr0_cic-preintegration/builds/296

Done by "Bai, ShanshanX" <shanshanx.bai@intel.com>

sysopenci reviewed (commented):
PR integrated into 'premerge' builder



More details, logs, and binaries: /absp/builders/celadon_pmr0_cic-premerge/builds/90

Done by "Bai, ShanshanX" <shanshanx.bai@intel.com>

sysopenci reviewed (commented):
cactus finished building and testing your changes 

It was requested for merge by "Bai, ShanshanX" <shanshanx.bai@intel.com> 

so we merge it. Thanks for your contribution.

more details, logs, and binaries: /absp/builders/celadon_pmr0_cic-mergerequest/builds/123
